core/capability-negotiation-3.2.md

```diff
@@ -42,7 +42,8 @@ If no other arguments are present, version 3.1 is assumed.
 ### Capabilities with values
 
 Servers MAY specify additional data for each capability using the
-`<name>[=<value>]` format.
+`<name>[=<value>]` format for `CAP LS` and `CAP NEW`.  `CAP ACK` and
+`CAP NAK` replies MUST NOT contain any values.
```

Comment:
Just not mention those at all, unless you happen to know about servers that go out of their way to have a `CAP ACK` payload that doesn't match what was `REQ`'d...